translations/fr.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Confirmed": "Confirmés",
+    "New Cases": "Nouveaux Cas",
+    "Deaths": "Décès",
+    "New Deaths": "Nouveaux Décès",
+    "Serious" : "Cas Sérieux",
+    "Recovered": "Gérisons",
+    "Active": "Actifs",
+    "Cases Per M" : "Cas Par M",
+    "Worldwide" : "Monde",
+    "statistic taken at ": "statistiques du ",
+    "N/A" : "N/A"
```

Comment:
@webac please add countries translations in order to be approved

Reply:
@bibaldo I just added countries.